src/api/apiTypes.js

```diff
@@ -81,7 +81,13 @@ export type Message = {
   content_type: 'text/html' | 'text/markdown',
   display_recipient: $FlowFixMe, // `string` for type stream, else PmRecipientUser[].
   edit_history: MessageEdit[],
-  flags: string[],
+
+  /**
+   * The first time a message is received, we store this data in separate sets of booleans
+   * within a hash table in state.flags. We don't interact with this data after that point,
+   * so it gets removed from state.messages in the Redux store.
+   */
```

Comment:
I'd expect prettier to have a way to automate line breaking for comments. I think it would still be nice to add `editor.rulers` to the workspace settings, though.

Reply:
Updated!